docs/guides/intro-to-tests.md

```diff
@@ -0,0 +1,154 @@
+# Introduction to tests
+
+## What does this cover
+
+This introduction is designed to give you a high-level overview on the basics of testing, which you'll need when you're tackling the problems in your coursework at Code Your Future.
+In this introduction to testing we'll cover the following things:
+
+- Code specification
+- What is a test ?
+- Anatomy of a test ( using Jest )
+- Understanding test feedback ( using Jest )
+
```

Comment:
I did have this in! - but then stripped it out because it was getting too technical. I think your points are far more concise and to the point though :)